task/git-batch-merge/0.2/git-batch-merge.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       default: "false"
     - name: gitInitImage
       description: The image used where the git-init binary is.
-      default: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.18.0"
```

Comment:
Is the bump of image version required in these two tasks? because I couldn't find the change in image in [v0.18.1](https://github.com/tektoncd/pipeline/releases/tag/v0.18.1) pipeline release.

Reply:
it's just about sanity.. it's not needed but it makes easy next time when we sed the version out.

Reply:
I still think it's a confusing user experience to bump these without bumping the version of the Tasks - when we do this it means that users can pull the same Task at the same version and get different behavior depending on when they pull it, see https://github.com/tektoncd/pipeline/pull/3258